src/hotspot/share/opto/loopopts.cpp

```diff
@@ -3895,6 +3898,9 @@ bool PhaseIdealLoop::partial_peel( IdealLoopTree *loop, Node_List &old_new ) {
     }
   }
 #endif
+
+  NOT_PRODUCT(C->print_method_iter(PHASE_PARTIAL_PEEL, 4, head);)
```

Comment:
Here you can also dump the new head `new_head_clone` after partial peeling.

Reply:
Thanks, updated.